src/components/table/DatasetTable.jsx

```diff
@@ -75,10 +90,23 @@ class DatasetTable extends React.PureComponent {
             <AddSVG className={classes.plusButton} />
           </NavLink>
         </div>
-        <JadeTable columns={columns} rows={rows} />
+        <JadeTable
+          columns={columns}
+          handleFilter={this.handleFilterDatasets}
+          rows={datasets}
+          summary={summary}
+          totalCount={datasetCount || datasets.length}
```

Comment:
I'd prefer if we just used the count, it should always be included from the API.